lighthouse-core/lib/emulation.js

```diff
@@ -95,13 +100,12 @@ async function emulate(driver, settings) {
   await driver.sendCommand('Network.enable');
   await driver.sendCommand('Network.setUserAgentOverride', {userAgent: params.userAgent});
 
-  if (!settings.internalDisableDeviceScreenEmulation) {
+  if (!params.internalDisableDeviceScreenEmulation) {
```

Comment:
also means fullpage might not work in devtools? +1 to the protocol method if I'm reading this right :)